src/index.ts

```diff
@@ -370,3 +371,26 @@ export const valid = (version: VersionInput) => {
 	}
 	return semver.valid(normalize(version));
 };
+
+/**
+ * @summary Return an incremented version
+ * @name inc
+ * @public
+ * @function
+ *
+ * @description Return the version incremented by the release type
+ * (major, premajor, minor, preminor, patch, prepatch, or prerelease), or null
+ * if it's not valid.
+ *
+ * @param {string|null|undefined} version
+ * @param {string} release
+ *
+ * @returns {string|null} - The incremented version string, or
+ * null if a valid semver string could not be found
+ */
+export const inc = (version: VersionInput, release: Release) => {
+	if (version == null) {
+		return null;
+	}
+	return semver.inc(normalize(version), release);
```

Comment:
It seems that `semver.inc()` also supports [additional optional parameters](https://github.com/npm/node-semver#prerelease-identifiers) (and the typings reveal even more). Should we support them as well?

Reply:
I just saw your comment, I'll open a new issue so that we don't forget about this

Reply:
https://github.com/resin-io-modules/resin-semver/issues/52